staff/daniel-toledo/precourse/calculadora/main.js

```diff
@@ -0,0 +1,169 @@
+
+
+function calculadora(num1,num2){
+
+	for(var i=0; i<arguments.length; i++){
+		if(typeof arguments[i]!='number'){
+			return 'Please insert at least 2 numbers';
+		}
+	}
+
+	function isInteger(num){
```

Comment:
Only as a little comment. I think JavaScript has a [method](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Number/isInteger) which does this.

For the rest, nice job!